src/hooks/useNotifications.test.ts

```diff
@@ -266,6 +281,263 @@ describe('hooks/useNotifications.ts', () => {
           result.current.notifications[0].notifications[4].subject.state,
         ).toBe('draft');
       });
+
+      it('should fetch discussion notifications with success - with colors', async () => {
+        const accounts: AuthState = {
+          ...mockAccounts,
+          enterpriseAccounts: [],
+          user: mockedUser,
+        };
+
+        const notifications = [
```

Comment:
I don't love having the stubbed data mid-test like this.

Reply:
Great feedback.  I've moved it out